internal/cli/root.go

```
@@ -124,3 +176,109 @@ func ParseGlobal(args []string) (Global, []string, error) {
+			if hasEquals {
+				// String-valued flags: stdlib stringValue.Set never errors.
+				_ = fs.Set(name, value)
+				i++
+				continue
+			}
+			// Consumes next token as value. Missing next token is a usage error
+			// — mirrors stdlib behavior for `flag needs an argument`.
+			if i+1 >= len(args) {
+				return Global{}, nil, fmt.Errorf("parse global flags: flag needs an argument: -%s", name)
+			}
+			_ = fs.Set(name, args[i+1])
```

golangci-lint (2.11.4)

[low] G602: slice index out of range

(gosec)